README.md

```diff
@@ -1,13 +1,24 @@
-Beautiful text chat for your community and website.
+## Scrollback, where communities hang out
 
-# Installation
+Scrollback gives you beautiful text chat for your community and website. [Try it out](https://scrollback.io/me)! 
 
-Run this command in terminal:
+You can even [embed Scrollback room](https://github.com/scrollback/scrollback/wiki/Basic-Usage#embed-scrollback-room) on your webpage/blog.
 
-    wget https://raw.githubusercontent.com/scrollback/scrollback/master/tools/install.sh -O install.sh && chmod +x install.sh && ./install.sh
+## Install
 
-To run Scrollback run:
+Scrollback is hosted Free Software but if you wanted to run it on your computer, you can! To install Scrollback locally:
 
-	sudo npm start
+1. Run `wget https://raw.githubusercontent.com/scrollback/scrollback/master/tools/install.sh -O install.sh && chmod +x install.sh && ./install.sh`.
```

Comment:
Do you want to explain briefly what the install script does? Sometimes people might be paranoid about this. For example:

It installs the dependencies (like Postgres 9.4, Redis, git, node.js) from your platform’s repositories after verifying that they aren’t already installed, initializes the database, builds the client-side JS and CSS and starts Scrollback. Tested on Arch, Debian, Ubuntu, Fedora and OSX.